Expensify/App
# Revert "Merge pull request #6268 from azimgd/fix-inverted-list"

This reverts commit 8d5a3ad56b0a3b94cf40fc699eb8e615d7547da7, reversing
changes made to 339bb6766da7cdffd0269b6c4144c3c110f5a02d.

cc @azimgd @thienlnam 

### Details
Fixes two deploy blockers caused by this pr https://github.com/Expensify/App/pull/6268 by reverting the pr

### Fixed Issues
$ https://github.com/Expensify/App/issues/6425
$ https://github.com/Expensify/App/issues/6436

### Tests
1. Open a new chat
2. Send a message
3. _Verify_ the message appears at the bottom of the screen instead of the top
4. Send a lot of messages
5. _Verify_ there are no visual issues when scrolling. Scrolling will be inverted.

### QA Steps
1. Open a new chat
2. Send a message
3. _Verify_ the message appears at the bottom of the screen instead of the top
4. Send a lot of messages
5. _Verify_ there are no visual issues when scrolling. Scrolling will be inverted.

### Tested On


![image](https://user-images.githubusercontent.com/22447860/143094500-1911827e-5aad-45ae-8f5c-96c0ec90dcf9.png)
